docs/src/pages/components/typography/typography.md

```diff
@@ -21,7 +21,7 @@ Roboto Font has a few easy ways to get started. For more advanced configuration,
 Shown below is a sample link markup used to load the Roboto font from a CDN:
 
 ```html
-<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" />
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500&display=swap" />
```

Comment:
It's one thing to change the documentation loading strategy, it's another to change the encouraging pattern. What does Google say about it? My concern with the swap mode is regarding the UI instability: all the text moves for a short period of time, even when the cache is filled.

Reply:
If you go to https://fonts.google.com/ then the autogenerated code will already include this param. 

As to developers.google:
> font-display is gaining adoption in many modern browsers. You can look forward to consistency in browser behavior as it becomes widely implemented.

-- https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/webfont-optimization#optimizing_loading_and_rendering

But from reading the `swap` section it seems like we shouldn't use this for normal text anyway. I didn't really pay attention to the technically details but now it seems like that the hype isn't justified if it should only be used for logo text.

Reply:
But then lighthouse audits penalize non-swap font-display.

Reply:
Thanks, it's good for me then 👍.